soc/xtensa/intel_adsp/common/CMakeLists.txt

```diff
@@ -100,6 +102,16 @@ add_custom_target(
       ${CMAKE_BINARY_DIR}/zephyr/main.mod 2>${NULL_FILE}
 )
 
+# west sign
+
+# FIXME: align with zephyr/cmake/mcuboot.cmake, maybe share some code
+
+add_custom_target(rimage_target ALL
+  DEPENDS ${CMAKE_BINARY_DIR}/zephyr/boot.mod ${CMAKE_BINARY_DIR}/zephyr/main.mod
+  COMMAND west sign --tool rimage --build-dir ${CMAKE_BINARY_DIR}
```

Comment:
In re-looking at this, I think this is missing a dependency on the west config file its using for params if I understand this right.

Reply:
I added it but it made no difference because linking happens always anyway (don't ask me why). Does it not for you?